posthog/session_recordings/session_recording_api.py

```diff
@@ -643,7 +659,8 @@ def snapshots(self, request: request.Request, **kwargs):
         if personal_api_key:
             SNAPSHOTS_BY_PERSONAL_API_KEY_COUNTER.labels(api_key=personal_api_key, source=source).inc()
 
-        is_v2_enabled = posthoganalytics.feature_enabled(
+        # we let people opt in to the API via query param or enforce it via feature flag
+        is_v2_enabled = allow_blob_v2 or posthoganalytics.feature_enabled(
```

Comment:
So, is it a "one-time try" approach then? Users having a query to allow blob_v2 can use it, but don't actually opt-in into it (as the feature flag isn't applied)?

Reply:
yep, my idea was to let them use the query param to test the new API so they can migrate their automation